### DISCUSSION TITLE
[QA] CREATE

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#263] [Q&A]
create

**Comment by NamgyungKim** [comment]
글 생성하고 아무런 액션이 없습니다.
글이 생성된건지 모르겠어요~~

**Reply by Hyevvy** [reply]
Blocker 때문에 멘트 고민중이라 지금 그렇게 된 거 같은데 원래는 alert가 떠용 !! 
그 부분 코드는 같이 올렸는데 alert하는게 없네용 ㅠ 현재 서버가 이상해서 확인이 불가하네욤 ㅠㅅㅠ

**Comment by NamgyungKim** [comment]
로그아웃 상태에서는 해당 페이지 에 접근하지 못하도록해야할 것 같습니다.

**Reply by Hyevvy** [reply]
create 에서 로그아웃 -> 메인페이지 redirect
edit 에서 로그아웃 -> detail 페이지로 redirect
 
================================

로그인이 안 된 유저가 create나 edit에 접근하면 이 때는 private router로 감싸기

**Comment by hyunjin0910** [comment]
(optional) 
두 글자 이상 써야 프로젝트가 생성되니까 textarea placeholder에 (두 글자이상) 써주는 것도 좋아보여요 ~

**Reply by Hyevvy** [reply]
아하 alert으로 할까 했는데 placeholder가 낫나 고민되네용 !! 감사합니당 ㅎㅎ
